frontend/src/components/admin/Admin.js

```diff
@@ -84,6 +86,20 @@ function Admin() {
               defaultMessage={"Common Properties"}
             />
           </SideNavLink>
+          <SideNavMenu
+            title={intl.formatMessage({ id: "admin.react" })}
+            renderIcon={CloudSatelliteConfig}
+          >
+            <SideNavMenuItem href="#configurationManagement">
```

Comment:
Why did you have to add multiple Links to the sampe page . 
We only need one . 
it should be labeled `Order Entry Configuration` as on the old UI

Reply:
These changes are according to grouping of same configurations which I proposed to you in one of the community meetings Right now we don't have all the page so I added <OrderEntryConfiguration/> as page for now !

I will update it no worries :slightly_smiling_face: